plugins/inputs/zipkin/zipkin_test.go

```diff
@@ -646,18 +645,18 @@ func postThriftData(datafile, address, contentType string) error {
 		return fmt.Errorf("could not read from data file %s", datafile)
 	}
 
-	req, err := http.NewRequest("POST", fmt.Sprintf("http://%s/api/v1/spans", address), bytes.NewReader(dat))
+	endpoint := fmt.Sprintf("http://%s/api/v1/spans", address)
+	req, err := http.NewRequest("POST", endpoint, bytes.NewReader(dat))
 	if err != nil {
-		return errors.New("HTTP request creation failed")
+		return fmt.Errorf("unable to create new POST request for: %q", endpoint)
```

Comment:
I think we owe the actual error text to the user. ;-)
```suggestion
		return fmt.Errorf("unable to create new POST request for %q: %w", endpoint, err)
```

Reply:
We do!